libs/journeys/ui/src/components/Video/VideoControls/VideoControls.tsx

```diff
@@ -229,6 +232,11 @@ export function VideoControls({
   ])
 
   function handlePlay(): void {
+    // Disables play button when video is at 0.0 for YouTube videos
+    if (isYoutube && state.playing && (player.currentTime() ?? 0) === 0) {
+      return
+    }
+
```

Comment:
make this a const isLoading 
`const isLoading = isYoutube && state.playing && (player.currentTime() ?? 0) === 0`

```suggestion
    // Disables play button when video is at 0.0 for YouTube videos
    if (isLoading) {
      return
    }

```

Reply:
Yes, this is needed. If I remove it, the small button is disabled but the screen button becomes usable

Reply:
Done